src/html.ts

```diff
@@ -3,7 +3,8 @@ import { FILE_MANIFEST, FILE_SW_REGISTER } from './constants'
 import { ResolvedVitePWAOptions } from './types'
 
 export function generateSWRegister(options: ResolvedVitePWAOptions) {
-  return `
+  return options.minify ? `"serviceWorker"in navigator&&window.addEventListener("load",()=>{navigator.serviceWorker.register("${join(options.basePath, options.filename)}",{scope:"${options.scope}"})});` : 
+`
```

Comment:
Don't think it's necessary here, can you omit this change?

Reply:
Done!